client/post-editor/media-modal/image-editor/index.jsx

```diff
@@ -73,9 +77,27 @@ const MediaModalImageEditor = React.createClass( {
 	onImageExtracted( blob ) {
 		const mimeType = MediaUtils.getMimeType( this.props.fileName );
 
+		// check if a title is already post-fixed with '(edited copy)'
+		const editedCopyText = this.props.translate(
+			'%(title)s (edited copy)', {
+				args: {
+					title: ''
+				}
+			} );
+		let title = this.props.title;
+		if ( title.indexOf( editedCopyText ) === -1 ) {
```

Comment:
Interesting approach. I wonder if there are localization concerns with other languages rearranging the words (e.g. maybe not always post-fixed). Even if that were the case, it would seem this should still work.